plugins/woocommerce-blocks/bin/webpack-configs.js

```diff
@@ -986,6 +978,151 @@ const getInteractivityAPIConfig = ( options = {} ) => {
 	};
 };
 
+const getCartAndCheckoutFrontendConfig = ( options = {} ) => {
+	let { fileSuffix } = options;
+	const { alias, resolvePlugins = [] } = options;
+	fileSuffix = fileSuffix ? `-${ fileSuffix }` : '';
+
+	const resolve = alias
+		? {
+				alias,
+				plugins: resolvePlugins,
+		  }
+		: {
+				plugins: resolvePlugins,
+		  };
+	return {
+		entry: getEntryConfig(
+			'cartAndCheckoutFrontend',
+			options.exclude || []
+		),
+		output: {
+			devtoolNamespace: 'wc',
+			path: path.resolve( __dirname, '../build/' ),
+			// This is a cache busting mechanism which ensures that the script is loaded via the browser with a ?ver=hash
+			// string. The hash is based on the built file contents.
+			// @see https://github.com/webpack/webpack/issues/2329
+			// Using the ?ver string is needed here so the filename does not change between builds. The WordPress
+			// i18n system relies on the hash of the filename, so changing that frequently would result in broken
+			// translations which we must avoid.
+			// @see https://github.com/Automattic/jetpack/pull/20926
```

Comment:
This solved it mainly for local dev, actual releases don't have this problem because translation is regenerated on each release.

Reply:
@senadir This bundle is basically copy/pasted from the front-end bundle including this comment, do you think I should remove this comment or adjust it?